sklearn/tree/_classes.py

```diff
@@ -883,6 +883,9 @@ class DecisionTreeClassifier(ClassifierMixin, BaseDecisionTree):
         :ref:`sphx_glr_auto_examples_tree_plot_unveil_tree_structure.py`
         for basic usage of these attributes.
 
+        .. versionchanged:: 1.4
```

Comment:
Not sure this makes sense to document a nested attributed change ...

The fact that nobody noticed/complained led me to believe that this is an implementation detail change that is not worth putting forward like this.

Reply:
Removed this then.